src/engine/neuron.h

```diff
@@ -3,25 +3,20 @@
 
 
 typedef enum {
+    NEURON_IF = 0,
+    NEURON_IF_REFRACT = 1,
+    NEURON_IF_ONE_SPIKE = 2,
+    NEURON_LIF = 3,
+    NEURON_LIF_REFRACT = 4,
+
+    NEURON_TYPES_CNT = 5,
     NEURON_INVALID,
-    NEURON_LIF,
```

Comment:
Add a comment referring to the arrays that are indexed by these, because it will be easier to know directly where we need to modify